deebot_client/hardware/deebot/9s1s80.py

```diff
@@ -164,17 +160,26 @@
         ),
         map=CapabilityMap(
             cached_info=CapabilityEvent(
-                CachedMapInfoEvent, [GetCachedMapInfo(version=2)]
+                CachedMapInfoEvent,
+                [deebot_client.commands.json.map.GetCachedMapInfo(version=2)],
```

Comment:
Please revert all unrelated changes and import the Commands instead of the module